### PR TITLE
Feat/hubspot integration UI

### PR DESCRIPTION
**Pull request by PatrikKozak**

### CODE REVIEW

File: src/components/Footer/index.tsx [hunk 1]
```diff
@@ -85,34 +188,39 @@ export const Footer: React.FC<FooterType> = props => {
...
             <Cell cols={5} colsM={6} colsS={8}>
               <p className={`${classes.colHeader} ${classes.thirdColumn}`}>Stay connected</p>
-
               <div>
-                <form
-                  method="POST"
-                  action="https://payloadcms.us18.list-manage.com/subscribe/post?u=f43c9eb62d4ce02e552a1fa9f&amp;id=e11798f237"
-                  onSubmit={() => analyticsEvent('newsletter')}
-                >
+                {error && <div>{`${error.status || '500'}: ${error.message || ''}`}</div>}
```
Comment (denolfe) [inline src/components/Footer/index.tsx:192]: Is there a need to expose this to the user?

File: src/components/Footer/index.tsx [hunk 2]
```diff
@@ -51,6 +85,75 @@ export const Footer: React.FC<FooterType> = props => {
...
+
+  const onSubmit = React.useCallback(() => {
+    setButtonClicked(false)
+    const submitForm = async () => {
+      setError(undefined)
+
+      try {
+        const formID = process.env.NEXT_PUBLIC_NEWSLETTER_FORM_ID
+        const hubspotCookie = getCookie('hubspotutk')
+        const pageUri = `${process.env.NEXT_PUBLIC_SITE_URL}${pathname}`
```
Comment (denolfe) [inline src/components/Footer/index.tsx:100]: Is there a slash between these from either the env var or usePathname?

File: src/components/Footer/index.tsx (hunk 1 above)
Comment (PatrikKozak) [reply src/components/Footer/index.tsx:192]: We are doing it with our regular forms as well. Not entirely necessary I suppose. What do you think?

File: src/components/CMSForm/index.tsx [hunk 3]
```diff
@@ -48,14 +51,25 @@ const RenderForm = ({ form }: { form: Form }) => {
         }, 1000)
 
         try {
+          const hubspotCookie = getCookie('hubspotutk')
+          const pageUri = `${process.env.NEXT_PUBLIC_SITE_URL}${pathname}`
+          const slugParts = pathname?.split('/')
+          const pageName =
+            slugParts && slugParts[slugParts.length - 1] === ''
```
Comment (denolfe) [inline src/components/CMSForm/index.tsx:58]: It feels like this logic could be cleaned up a bit. Something like `slugParts?.at(-1) === ''`

File: src/components/Footer/index.tsx (hunk 2 above)
Comment (PatrikKozak) [reply src/components/Footer/index.tsx:100]: `pathname` returns with a slash, yes.

**Pull request merged**